server/scan/device_handling.py

```
@@ -1,764 +1,816 @@
                         return vendor
+        mylog('debug', f"[Vendor Check] No vendor found for '{pMAC}'")
+        return "-1"
+    except FileNotFoundError:
+        mylog('error', f"[Vendor Check] Vendors file {file_path} not found.")
+        return "-1"
+    except Exception as e:
+        mylog('error', f"[Vendor Check] Error reading vendors file: {str(e)}")
+        return "-1"
+
+# Guess device attributes such as type of device and associated device icon
+def guess_device_attributes(
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)